script.js

```diff
@@ -9,6 +9,11 @@ const weatherIcon = document.querySelector(".weather-icon");
 async function checkWeather(city) {
   const response = await fetch(apiUrl + city + `&appid=${apiKey}`);
 
+  if (city.length < 3) {
+    window.alert("City name much be at least 3 characters long.");
+    searchBox.classList.add("error-input");
+    return;
+  }
```

Comment:
Can you move this logic, where your button is being triggered.

Logic is what we want, but is on the incorrect place.